src/widgets/visit/edit-visit-component.tsx

```diff
@@ -112,7 +112,7 @@ type EditVisitProps = {
 
 type PatientVisitType = {
   encounters: any[];
-  location: any;
+  location?: any;
```

Comment:
Can we type these `any`s? What is the type of the `encounters` array entries? What's the type of `location`? Thanks!